README.rst

```diff
@@ -45,23 +45,98 @@ files.
 Quick Guide
```

Comment:
I don't know if you care about which case you use for headings, but in Plone we use sentence-casing. No one knows the formal English rules by heart, and it is much easier to remember to only upper case the first letter of the first word in a heading, with exceptions for acronyms and proper nouns.

Whatever you choose, consistency is a good idea. I see different heading casing rules in this file. 

```suggestion
Quick start guide
```